GDAXClient/Services/Products/Models/CandleGranularity.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace GDAXClient.Services.Products.Models
+{
+    /// <summary>
```

Comment:
please remove the summary comments here. I try to keep the code clean of this as this information is available in the docs